code/modules/mob/living/carbon/human/human.dm

```diff
@@ -171,15 +171,15 @@
 		return 1
 	return 0
 
-/mob/living/carbon/human/var/co2overloadtime = null
+/mob/living/carbon/human/var/co2overloadtime
 /mob/living/carbon/human/var/temperature_resistance = T0C+75
```

Comment:
This is awful, for future reference this should be on human defines but not relevant to pr